src/app/login/login.page.ts

```diff
@@ -16,7 +17,9 @@ export class LoginPage implements OnInit {
 
   public maungSource = MaungImage;
 
-  constructor(public fetchService: FetchService) {}
+  constructor(
+    public fetchService: FetchService,
```

Comment:
Nanti fetch servicenya bisa diapus sih. Kayaknya pake `this.http` aja kyk bapaknya tadi